client/src/pages/Login/useStyles.ts

```diff
@@ -11,16 +16,21 @@ const useStyles = makeStyles(() => ({
     display: 'flex',
     alignItems: 'flex-start',
     justifyContent: 'space-between',
+    justifyItems: 'center',
     flexDirection: 'column',
-    minHeight: '100vh',
-    paddingTop: 23,
+  },
+  titleWrapper: {
+    padding: 80,
+    paddingTop: 60,
+    height: '600px',
```

Comment:
there's inconsistency between pixels and just integers (by default it'll render as pixels)

Reply:
Thanks for pointing all of these out! :)
As I'm trying to overwrite my bad(?) habits from react native, it's nice to be reminded of some of these big ones.

Reply:
I've made a few changes based on what you've pointed out.  Let me know if I've missed anything or if you have any questions.